index.html

```diff
@@ -478,6 +478,10 @@ <h2>Glossary</h2>
 
 <p><dfn  class="lint-ignore export">User-supplied value</dfn>. Unreserved [=syntactic content=] in a [=vocabulary=] that is assigned by users, as distinct from reserved keywords in a given format or protocol. Users generally expect that their user-supplied values can be words or phrases in their preferred [=natural language=]. This is why [[CHARMOD]] recommends that "Specifications <em class="rfc2119">SHOULD NOT</em> arbitrarily exclude code points from the full range of Unicode code points from <span class="uname">U+0000</span> to <span class="uname">U+10FFFF</span> inclusive." [[CHARMOD-NORM]] gives <a href="https://www.w3.org/TR/charmod-norm/#dfn-user-supplied-value">some examples</a>. </p>
 
+<p><dfn class="lint-ignore">UTF-8</dfn>. <a href="https://unicode.org/glossary/#UTF_8">Unicode definition</a>: <q>A multibyte encoding for text that represents each Unicode character with 1 to 4 bytes, and which is backward-compatible with ASCII. UTF-8 is the predominant form of [=Unicode=] in web pages. More technically: (1) The <a href="https://unicode.org/glossary/#utf_8_encoding_form"> UTF-8 encoding form</a>. (2) The <a href="https://unicode.org/glossary/#utf_8_encoding_scheme"> UTF-8 encoding scheme</a>. (3) “UCS Transformation Format 8,” defined in Annex D of ISO/IEC 10646:2003, technically equivalent to the definitions in the Unicode Standard.</q></p>
```

Comment:
I thought we were going to make external links target blank?

Reply:
According to https://github.com/w3c/i18n-glossary/pull/52#discussion_r1279062427 I'll create a new issue for that (it's on my to-do list).

Reply:
Done: https://github.com/w3c/i18n-glossary/issues/53